orchagent/pfcwdorch.cpp

```diff
@@ -600,11 +611,8 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::unregisterFromWdDb(const Port& po
 
         // Clean up
         RedisClient redisClient(PfcWdOrch<DropHandler, ForwardHandler>::getCountersDb().get());
-        string countersKey = COUNTERS_TABLE ":" + sai_serialize_object_id(queueId);
-        redisClient.hdel(countersKey, "PFC_WD_DETECTION_TIME");
-        redisClient.hdel(countersKey, "PFC_WD_RESTORATION_TIME");
-        redisClient.hdel(countersKey, "PFC_WD_ACTION");
-        redisClient.hdel(countersKey, "PFC_WD_STATUS");
+        string countersKey = PfcWdSwOrch<DropHandler, ForwardHandler>::getCountersTable()->getTableName() + PfcWdSwOrch<DropHandler, ForwardHandler>::getCountersTable()->getTableNameSeparator() + sai_serialize_object_id(queueId);
```

Comment:
>PfcWdSwOrch<DropHandler, ForwardHandler>:: [](start = 29, length = 42)

I think you can remove this specifier.

Reply:
pfcwdorch.cpp:616:47: error: there are no arguments to 'getCountersTable' that depend on a template parameter, so a declaration of 'getCountersTable' must be available [-fpermissive]
         string countersKey = getCountersTable()->getTableName() + PfcWdSwOrch<DropHandler, ForwardHandler>::getCountersTable()->getTableNameSeparator() + sai_serialize_object_id(queueId);
                                               ^
Compiler does not like the omission

pfcwdorch.cpp:616:47: note: (if you use '-fpermissive', G++ will accept your code, but allowing the use of an undeclared name is deprecated)
Makefile:841: recipe for target 'orchagent-pfcwdorch.o' failed
make[2]: *** [orchagent-pfcwdorch.o] Error 1